pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +79,55 @@ Our projects on Python Discord are open source and [available on Github](https:/
   </div>
 </div>
 
-If you don't understand anything or need clarification, feel free to ask any staff member with the  **@PyDis Core Developers** role in the server. We're always happy to help!
+# How do I start contributing?
+  Unsure of what contributing to open source projects involves? Have questions about how to use GitHub? Just need to know about our contribution etiquette? Completing these steps will have you ready to make your first contribution no matter your starting point.
+
+  Feel free to skip any steps you're already familiar with, but please make sure not to miss the  [Contributing Guidelines](#5-read-our-contributing-guidelines).
```

Comment:
There's a double-space here before the link

Reply:
Fixed